sklearn/metrics/base.py

```diff
@@ -124,3 +125,72 @@ def _average_binary_score(binary_metric, y_true, y_score, average,
         return np.average(score, weights=average_weight)
     else:
         return score
+
+
+def _average_multiclass_ovo_score(
+        binary_metric, y_true, y_score, labels=None, average='macro'):
+    """Uses the binary metric for one-vs-one multiclass classification,
+    where the score is computed according to the Hand & Till (2001) algorithm.
+
+    Parameters
+    ----------
```

Comment:
Ideally the list should follow the same order as the parameters passed to `__init__`